README.md

```diff
@@ -4,6 +4,37 @@
 
 This gem is an implementation of the [language server protocol specification](https://microsoft.github.io/language-server-protocol/) for Ruby, used to improve editor features.
 
+# Overview
+
+The intention of Ruby LSP is to provide a fast, accurate, and robust coding environment for Ruby developers, especially for large codebases.
+
+It's part of a [wider Shopify goal](https://github.com/Shopify/vscode-shopify-ruby) to provide a state-of-the-art experience to Ruby developers using modern standards for cross-editor features, documentation and debugging.
+
+It provides many features, including:
+
+* Syntax highlighting
+* Linting and formatting
+* Code folding
+* Selection ranges
+
+When used on typed Sorbet code, additional features are available:
```

Comment:
The Ruby LSP and Sorbet work completely independently - with no interaction between the two whatsoever. I don't think we should include this section, we can simply say "The Ruby LSP is designed to work by itself or paired with Sorbet".

Reply:
Ah, hover didn't seem to be working without Sorbet when I tried previously so I thought there was some connection, but it's working now so something separate must have been happening.